youtube.css

```
@@ -0,0 +1,155 @@
+html {
+  border: 2px solid gray;
+  margin-left: auto;
+  margin-right: auto;
+  width: 1500px;
```

Try to make this smaller and responsive.
```suggestion
  min-width: 1200px; /* from the archived page properties */
```